docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
+# Release Documentation
+
+# Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+# Basic Terms
+
+## Stability
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api).
+- [Command line options](api.md#command-line-interface).
+- [Device Model](device_model.md).
+- Device tree, device list, ACPI, Hyper-V enlightenments and any other features
+  exposed to guest.
+- KVM compatibility.
+- UEFI compatibility.
+- Rust edition compatibility.
+
+This list might be by far incomplete. Still this document strives to gather any
+possible indicators to be desperately considered relevant for the stability context.
+
+## Experimental features
+
+Features that are being actively developed and are possibly incomplete or known to be
+unstable are considered experimental. These features are in no case recommended for
+use in production.
+
+Features currently considered experimental:
+
+- TDX.
+- vfio-user.
+- vDPA.
+
+## Security
+
+Security issues usually involve only defects directly related to Cloud Hypervisor.
+
+While security issues in the compiler or third party components are out of scope as
+they can't be influenced by the Cloud Hypervisor project, still fixes to affected
+code or changes to ensure a non vulnerable dependency version is used might trigger
+a new point release. However, these measures won't affect possibly already released
+tarballs or binary artifacts.
+
+A new CVE needs to be requested for every security flaw.
+
+A way to reproduce a security issue might not be made public.
+
+Any downstream projects including or packaging Cloud Hypervisor (for example Linux
+distributions) should ideally have a way to be notified on the security issue prior
+the release. Security issuse are to be explicitly mentioned as such in the relase
+notes including the CVE if available.
+
+# Releases
+
+## Versioning
+
+The versioning scheme uses `MAJOR.POINT` as pattern, this means:
+
+- `MAJOR` can introduce incompatible changes. Changes to the [API](api.md#rest-api),
+  [CLI options](api.md#command-line-interface) and [device model](device_model.md)
+  require a premonition at least 2 releases in advance for the actual change to take
+  place.
+- `POINT` can contain bug and security fixes.
+
+## Regular Release Cadence
+
+Cloud Hypervisor is under active development.
+
+Approximately every 6 weeks a new major release is issued.
+
+Point releases are issued on demand, when important bugfixes are in the queue. A
```

Comment:
Nit: may want to put these contents into one paragraph.

Reply:
Done that way. Thanks